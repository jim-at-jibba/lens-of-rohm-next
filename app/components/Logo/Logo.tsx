import Heading from "../Heading/Heading"

const Logo = () => {
  return (
    <Heading level={1} fontSize={40} m={4}>
      LENS OF ROHM
    </Heading>
  )
}

export default Logo
