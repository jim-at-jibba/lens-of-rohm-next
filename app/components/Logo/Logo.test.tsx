import {render, cleanup} from "@testing-library/react"
import Logo from "./Logo"

afterEach(cleanup)

describe("Logo Component", () => {
  it("should render properly", () => {
    const logo = render(<Logo />)
    expect(logo).toMatchSnapshot()
  })

  it("should contain my name", () => {
    const {getByTestId} = render(<Logo />)
    expect(getByTestId("logo")).toHaveTextContent("LENS OF ROHM")
  })
})
