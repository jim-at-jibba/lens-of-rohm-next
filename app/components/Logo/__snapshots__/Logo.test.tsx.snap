// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should render properly 1`] = `undefined`;
